### PR TITLE
[ROSBE][UNIX] Accept Yes/No answers with upper case letters

### PR DESCRIPTION
**Pull request by GeoB99**
As the commit message says, answering with upper case letters leads the script quitting. This patch should correct this behaviour.

### CODE REVIEW

**Review by HeisSpiter — CHANGES_REQUESTED** [review]
Can you uppercase instead of doing various comparisons?
(using bash syntax - if we accept being only bash 4 compliant, tr, awk, whatever method you prefer).

**Comment by GeoB99** [comment]
@HeisSpiter I decided to use `tr` for uppercase conversion. Is it good now?

File: RosBE-Unix/Base-i386/RosBE-Builder.sh [hunk 1]
```diff
@@ -177,6 +177,7 @@ if [ "$1" = "" ]; then
 				else
 					echo "The directory \"$installdir\" is not empty. Do you really want to continue? (yes/no)"
 					read -p "[no] " answer
+					answer = $( echo "$answer" | tr '[a-z]' '[A-Z]' )
 					echo
 
 					if [ "$answer" != "yes" ]; then
```
Comment (HeisSpiter) [inline RosBE-Unix/Base-i386/RosBE-Builder.sh]: This won't work.

File: RosBE-Unix/Base-i386/scripts/rosbelibrary.sh [hunk 2]
```diff
@@ -37,10 +37,11 @@ check_root()
 
 		local answer
 		read -p "[no] " answer
+		answer = $( echo "$answer" | tr '[a-z]' '[A-Z]' )
 
 		if [ "$answer" = "yes" ]; then
```
Comment (HeisSpiter) [inline RosBE-Unix/Base-i386/scripts/rosbelibrary.sh]: This won't work.

File: RosBE-Unix/Base-i386/scripts/rosbelibrary.sh [hunk 3]
```diff
@@ -37,10 +37,11 @@ check_root()
...
 			echo
-		else
+		elif [ "$answer" = "no" ]; then
```
Comment (HeisSpiter) [inline RosBE-Unix/Base-i386/scripts/rosbelibrary.sh]: I'd rather let this change out.
It's not just about changing caps here.

**Review by HeisSpiter — CHANGES_REQUESTED** [review]
Nota: test your changes before pushing the "push" button ;-).
`$ answer="yes"; echo "$answer" | tr '[a-z]' '[A-Z]'`
`YES`

**Comment by GeoB99** [comment]
@HeisSpiter Would something like this be tolerable?
```Bash
if [[ "$answer" = [yY][eE][sS] ]]; then
```
I further tested the Bash script and the uppercase letters are accepted in the input as you can see below (yes and YES also do work). I dropped the `tr` method because it always gave me `command not found` and syntax errors (this is the first time I touch Bash script).

```
Do you really want to continue? (yes/no)
[no] Yes

Checking for the needed tools...
Checking for as... OK
Checking for bison... OK
Checking for flex... OK
Checking for gcc... OK
Checking for g++... OK
Checking for grep... OK
Checking for makeinfo... OK
Checking for python... OK
Checking for GNU Make... OK
Checking for pkg-config... OK
Checking for zlib... OK

Installation Directory
In which directory do you want to install it?
Enter the path to the directory here or simply press ENTER to install it into the default directory.
[/usr/local/RosBE] 
```

**Comment by learn-more** [comment]
@HeisSpiter How about these changes?

**Pull request merged**